lib/license_scout/license.rb

```diff
@@ -107,7 +107,7 @@ def license_content(license_id, contents_url)
         begin
           LicenseScout::Log.debug("[license] Pulling license content for #{license_id} from #{new_url}")
           open(new_url).read
-        rescue OpenURI::HTTPError, Errno::ECONNREFUSED, SocketError
+        rescue
```

Comment:
This is going to break the redirection logic in the rescue block below.

Reply:
gah, you're right